frontend/templates/ng-templates/login.html

```diff
@@ -1,27 +1,57 @@
 <!-- login.html -->
 <div ng-controller="mainController as login" class="container">
 
-    <div class="login-header">
-        <h1>Login to swefreq is changing</h1>
+    <div class="alert alert-success" style="width:75%; margin: 20px auto;" ng-if="login.loginType == 'updated'">
```

Comment:
Don't use `style` attributes. If you want to limit the width use the grid system from bootstrap: https://getbootstrap.com/docs/3.3/css/#grid (`class="col-md-9"`).